mlem/contrib/scipy.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any, ClassVar, Iterator, Tuple
+
+import scipy
+from scipy import sparse
+from scipy.sparse import csr_matrix
+
+from mlem.core.artifacts import Artifacts, Storage
+from mlem.core.data_type import (
+    DT,
+    DataHook,
+    DataReader,
+    DataType,
+    DataWriter,
+    WithDefaultSerializer,
+)
+from mlem.core.hooks import IsInstanceHookMixin
+from mlem.core.requirements import InstallableRequirement, Requirements
+
+
+class ScipySparceMatrix(
+    WithDefaultSerializer, DataType, DataHook, IsInstanceHookMixin
+):
+    type: ClassVar[str] = "csr_matrix"
```

Comment:
I suppose this could be extended to cover other sparse matrices https://github.com/iterative/mlem/issues/540